pom.xml

```diff
@@ -43,6 +43,14 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
i think you said we don't actually need this, is that right?  i'm not sure if this will cause problems in some of the other components.  i don't know if everything that uses ice4j has moved to java 1.8

Reply:
We deliberately switched to Java 8 and removed pre-1.8 hacks. I don't see any reason to go back to Java 1.7. Hell, even 1.8 is out of support from Oracle in two months from now.

Reply:
@ibauersachs just rolled back to 1.7 :( Would you mind me adding 1.8 again ?

Reply:
Yes, please just remove your last two commits.

Reply:
@ibauersachs done, thank you very much for reviewing this PR!